QuickFIXn/QuickFix.csproj

```
@@ -43,4 +43,8 @@
       <_Parameter1>UnitTests</_Parameter1>
     </AssemblyAttribute>
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.2" />
```

I think this could specify ("at least") version 6.0.0, as it doesn't have any [dependencies](https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/6.0.0) whereas the later versions do. This should reduce any qualms about taking the dependency. We are only using the interfaces here

```suggestion
    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
```